package/src/worker_types.ts

```diff
@@ -29,6 +29,11 @@ export type DownsamplingWorkerResponseReady = {
   command: 'ds-ready';
 };
 
+export type PorcupineWorkerResponseFailed = {
```

Comment:
I'm guessing this should be `DownsamplingWorkerResponseFailed`?

Reply:
good catch!